coalib/settings/ConfigurationGathering.py

```diff
@@ -394,13 +394,14 @@ def gather_configuration(acquire_settings,
         arg_list = sys.argv[1:] if arg_list is None else arg_list
     sections, targets = load_configuration(arg_list, log_printer, arg_parser,
                                            args=args)
-    local_bears, global_bears = fill_settings(sections,
-                                              acquire_settings,
-                                              log_printer)
+    local_bears, global_bears, aspects = fill_settings(sections,
+                                                       acquire_settings,
+                                                       log_printer)
     save_sections(sections)
     warn_nonexistent_targets(targets, sections, log_printer)
 
     return (sections,
             local_bears,
             global_bears,
+            aspects,
```

Comment:
This is a breaking change to the public coala API.
A very large one.  coala-quickstart is broken by this.
I dont recall this being in an earlier patch I reviewed.